tenseal/tensors/bfvvector.cpp

```diff
@@ -171,6 +171,11 @@ BFVVector& BFVVector::mul_inplace(BFVVector to_mul) {
 
     this->context->evaluator->multiply_inplace(this->ciphertext,
                                                to_mul.ciphertext);
+                                               
+    // TODO: make this optional
+    // relineraization after ciphertext-ciphertext multiplication
+    this->context->evaluator->relinearize_inplace(this->ciphertext,
+                                                  this->context->relin_keys());
```

Comment:
Maybe it would be nice to add some tests that verify the size of the ciphertext after the bfvvector multiplication, to underline the importance of the fix.

Reply:
I love this, we could add for both BFV and CKKS!

Reply:
Yeah, I think we should add these tests.